pkg/operator/ceph/object/bucket.go

```diff
@@ -77,6 +77,11 @@ func GetBucketStats(c *Context, bucketName string) (*ObjectBucketStats, bool, er
 		"stats",
 		"--bucket", bucketName)
 
+	errTxt := ""
+	if err != nil {
+		errTxt = err.Error()
+	}
+	logger.Debugf("BUCKET STATS. result=%q, err=%q", result, errTxt)
```

Comment:
Why not simply printing `err` with  `%v`?

Reply:
I'm planning to remove this change after troubleshooting the CI